website/docs/docs/build/sl-getting-started.md

```diff
@@ -4,23 +4,33 @@ title: Get started with MetricFlow
 description: "Learn how to create your first semantic model and metric."
 sidebar_label: Get started with MetricFlow
 tags: [Metrics, Semantic Layer]
+meta:
+  api_name: dbt Semantic Layer API
 ---
 
-This getting started page recommends a workflow to help you get started creating your first metrics. Here are the following steps you'll take:
+This getting started page recommends a workflow to help you create your first metrics. This guide uses the [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template). If you prefer using a sandbox project to follow the guide, you can clone the repo from GitHub. Otherwise, you can create semantic models and metrics for your own dbt project. 
```

Comment:
It might be beneficial to mention somewhere up top that the Semantic Layer items are Cloud only (a sort of subtle upsell).